<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewQQ</class>
 <widget class="QWidget" name="NewQQ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NewQQ</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,12,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="hTitle"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hcenterWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hStatus"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
